sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/v2/RefreshTableExec.scala

```diff
@@ -17,15 +17,24 @@
 
 package org.apache.spark.sql.execution.datasources.v2
 
+import org.apache.spark.sql.SparkSession
 import org.apache.spark.sql.catalyst.InternalRow
 import org.apache.spark.sql.catalyst.expressions.Attribute
-import org.apache.spark.sql.connector.catalog.{Identifier, TableCatalog}
+import org.apache.spark.sql.connector.catalog.{Identifier, Table, TableCatalog}
 
 case class RefreshTableExec(
+    session: SparkSession,
     catalog: TableCatalog,
+    table: Table,
     ident: Identifier) extends V2CommandExec {
   override protected def run(): Seq[InternalRow] = {
     catalog.invalidateTable(ident)
+
+    // invalidate all caches referencing the given table
+    // TODO: re-cache the table itself once we support caching a DSv2 table
```

Comment:
Hi, @sunchao . We recommend to have IDed TODO like `TODO(SPARK-12345)`.

Reply:
If you have JIRA id for that, please use it here. Otherwise, please make it first.

Reply:
Cool thanks for the info. I don't think there is one yet so I'll make one now.